doc/manual/src/language/operators.md

```diff
@@ -159,7 +159,7 @@ All comparison operators are implemented in terms of `<`, and the following equi
 
 | comparison   | implementation        |
 |--------------|-----------------------|
-| *a* `<=` *b* | `! (` *b* `<` *a* `)` |
```

Comment:
The current text looks correct to me. Maybe you expected `a` and `b` not to be swapped.

I think they were swapped to reflect that they're not just equivalences, but implementation rules as mentioned in the sentence before the table.

Reply:
Oh true, didn't seen that was swapped